src/views/cards/Cards/index.tsx

```diff
@@ -30,6 +42,16 @@ export default function Cards() {
               }}
               size="small"
               color={color}
+              onClick={() =>
+                handleMoveToEditCardPage({
+                  id,
+                  cardName,
+                  cardNumber,
+                  cardOwnerName,
+                  expireDate,
+                  color,
+                })
+              }
```

Comment:
```suggestion
              onClick={() =>
                handleMoveToEditCardPage({
                  id,
                  color,
                  ...cardProps
                })
              }
```
map에 인자를 적어줄 때 id랑 color 이외 속성들은 한번에 묶어서 받으면 어떨까요?